docs/book/src/testing/unit-testing.md

```diff
@@ -53,7 +53,44 @@ Tests with `#[test(should_revert)]` considered to be passing if they are reverti
 
 ## Calling Contracts
 
-***Coming Soon***
+Unit tests can call contract functions an example for such calls can be seen below.
```

Comment:
```suggestion
Unit tests can call contract functions, like so:
```

Reply:
Nice suggestions thanks a lot! I will have a follow-up PR anyways (for injecting `CONTRACT_ID` automatically) will mark these resolved for now and fix them in the follow-up not to dismiss josh's review.